easybuild/easyblocks/t/tensorflow.py

```diff
@@ -961,16 +961,20 @@ def build_step(self):
             + ['build']
             + self.target_opts
             + [self.cfg['buildopts']]
-            # specify target of the build command as last argument
-            + ['//tensorflow/tools/pip_package:build_pip_package']
         )
+        if LooseVersion(self.version) < LooseVersion('2.16'):
```

Comment:
Just FYI: `LooseVersion(self.version) < '2.16':` is equivalent, if you prefer that.